src/coprocessor/dag/batch/executors/table_scan_executor.rs

```diff
@@ -328,16 +328,16 @@ mod tests {
 
     /// Test Helper for normal test with fixed schema and data.
     /// Table Schema: ID (INT, PK), Foo (INT), Bar (FLOAT, Default 4.5)
-    /// Column id:    1,            2,         4
-    /// Column offset:   0,            1          2
-    /// Table Data:  (1,            Some(10),  Some(5.2)),
-    ///              (3,            Some(-5),   None),
-    ///              (4,             None,      default(4.5)),
-    ///              (5,             None,      Some(0.1)),
-    ///              (6,             None,      default(4.5)),
+    /// Column id:    1,             2,             4
+    /// Column offset: 0,            1,             2
```

Comment:
The '0' in this line is not well aligned. It's now neither at the beginning or at the center. Better to add one more space to other lines.

Reply:
Sharp eye!